contracts/strategies/88mph/EightyEightMphStrategyMainnet_UNI.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity 0.5.16;
+
+import "./base/EightyEightMphStrategy.sol";
+
+contract EightyEightMphStrategyMainnet_UNI is EightyEightMphStrategy {
+
+  address public eightyEightyMphUni_unused; // just a differentiator for the bytecode
+
+  constructor() public {}
+
+  function initializeStrategy(
+    address _storage,
+    address _vault
+  ) public initializer {
+    address underlying = address(0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984); // UNI
+    address rewardPool = address(0x5dda04b2BDBBc3FcFb9B60cd9eBFd1b27f1A4fE3);
+    address mph = address(0x8888801aF4d980682e47f1A9036e589479e835C5);
+    address weth = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+
+    bytes32 sushiDex = bytes32(0xcb2d20206d906069351c89a2cb7cdbd96c71998717cd5a82e724d955b654f67a);
+    bytes32 uniV3Dex = bytes32(0x8f78a54cb77f4634a5bf3dd452ed6a2e33432c73821be59208661199511cd94f);
+
+    EightyEightMphStrategy.initializeBaseStrategy(
+      _storage,
+      _vault,
+      underlying,
+      rewardPool,
+      address(0), // pot pool -> manually set it later
+      500, // percentage of rewards after profit fee to distribute as xMPH,
+      180 days // maturation target
```

Comment:
This would be the default maturation target for the first strategy deployment. 

We have been notified by 88mph that they will rework their staking stystem (turn xMPH into some updated version called veMPH). This will require an easily manageable update in this strategy and the easiest should be a migration to a new strategy. Accounting for everything that might happen and ensuring it will be upgradeable could be a bit tricky... Not sure if this is a problem? 

According to the information we got from 88mph we should be good with the old xMPH for another 6months+.